components/item-divider/ItemDivider.tsx

```diff
@@ -0,0 +1,17 @@
+import classes from '*.module.css';
+
+import { makeStyles } from '@material-ui/core';
+
+const useStyles = makeStyles({
+  divider: {
+    background: 'black',
+    width: 'calc(100% + var(--right-column-width))',
+    height: '1px',
+    margin: '2rem 0',
```

Comment:
hope you didn't put much effort into this. Because we will change stuff anyways when we do the styling.

Reply:
Sure. These stylings are just to give a general idea how to implement certain stylings. (e.g. in which components to put them). Although in this case, the divider looks pretty much the way it does on the mockups.